include/deal2lkit/utilities.h

```diff
@@ -460,4 +460,26 @@ void smart_delete (SmartPointer<TYPE> &sp)
     }
 }
 
+/**
+ *  A simple class to shift a vector by a scalar.
+ *  This function is deprecated in deal but needed in many codes
+ */
+
+template <typename VEC>
+void vector_shift(VEC &in_vec, double a_scalar)
+{
+#ifdef DEAL_II_WITH_CXX11
+  for (auto i : in_vec.locally_owned_elements())
+    {
+      in_vec[i] += a_scalar;
+    }
+#else
+  for (IndexSet::Iterator it=in_vec.locally_owned_elements().begin(); it != in_vec.locally_owned_elements().end(); ++it)
```

Comment:
I think this should be **ElementIterator**

Since this is identical in every aspect to the above (they have the same machine code), can you just remove the CXX_11 guard and use only this version?